src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_RESIZE_MULTIPLIER = 2;
+    private int size;
+    private int capacity;
+    private int threshold;
```

Comment:
useless variable, u can calculate it when `resizeIfNeeded` performed and no need to store it